contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -32,6 +32,7 @@ contract PrizeStrategy is PrizeStrategyStorage,
 
   using SafeMath for uint256;
```

Comment:
There are at least 5 equations in this file not using SafeMath

Reply:
Ah!  That must be fixed

Reply:
Fixed using SafeMath & SafeCast for uint256